www/components/ThemeModal.js

```diff
@@ -0,0 +1,21 @@
+import styles from '../styles/ThemeModal.module.css';
+
+import placeholder from '../public/placeholderTheme.png';
+import ThemeCard from './ThemeCard';
+
```

Comment:
Can we make this a general `Modal` component for repeated use? I bet we will want to use something like this for the user profile.